### PR TITLE
Update Visa calculator for Saudi Arabia and Bahrain

### PR DESCRIPTION
**Pull request by 1pretz1**
Trello:
https://trello.com/c/WeCrtlRX/1222-check-if-you-need-a-uk-visa-visa-waiver-changes

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: app/flows/check_uk_visa_flow/outcomes/outcome_medical_y.erb
Comment (mattclear) [inline app/flows/check_uk_visa_flow/outcomes/outcome_medical_y.erb:13]: Change [EVW] to [electronic visa waiver (EVW)] to make the link text more accessible.

File: app/flows/check_uk_visa_flow/outcomes/outcome_school_y.erb
Comment (mattclear) [inline app/flows/check_uk_visa_flow/outcomes/outcome_school_y.erb:13]: Change [EVW] to [electronic visa waiver] to make the link text more accessible.

File: app/flows/check_uk_visa_flow/outcomes/outcome_standard_visitor_visa.erb
Comment (mattclear) [inline app/flows/check_uk_visa_flow/outcomes/outcome_standard_visitor_visa.erb:13]: Change [EVW] to [electronic visa waiver] to make the link text more accessible.

File: app/flows/check_uk_visa_flow/outcomes/outcome_study_m.erb
Comment (mattclear) [inline app/flows/check_uk_visa_flow/outcomes/outcome_study_m.erb:15]: Change [EVW] to [electronic visa waiver] to make the link text more accessible.

File: app/flows/check_uk_visa_flow/outcomes/outcome_tourism_visa_partner.erb
Comment (mattclear) [inline app/flows/check_uk_visa_flow/outcomes/outcome_tourism_visa_partner.erb:13]: Change [EVW] to [electronic visa waiver] to make the link text more accessible.

File: app/flows/check_uk_visa_flow/outcomes/outcome_transit_leaving_airport.erb
Comment (mattclear) [inline app/flows/check_uk_visa_flow/outcomes/outcome_transit_leaving_airport.erb:13]: Change:

an [EVW] 

to:

+ an [electronic visa waiver] 

Reason:

To make the link text more accessible. Also bullet markdown is missing.

File: app/flows/check_uk_visa_flow/outcomes/outcome_transit_leaving_airport.erb
Comment (mattclear) [inline app/flows/check_uk_visa_flow/outcomes/outcome_transit_leaving_airport.erb:18]: Change [EVW] to [electronic visa waiver] to make the link text more accessible.

File: app/flows/check_uk_visa_flow/outcomes/outcome_transit_to_the_republic_of_ireland.erb
Comment (mattclear) [inline app/flows/check_uk_visa_flow/outcomes/outcome_transit_to_the_republic_of_ireland.erb:25]: Change [EVW] to [electronic visa waiver] to make the link text more accessible.

File: app/flows/check_uk_visa_flow/outcomes/outcome_work_m.erb
Comment (mattclear) [inline app/flows/check_uk_visa_flow/outcomes/outcome_work_m.erb:29]: Change lines 27-29 from:

^
You will not need a visa if you hold a valid [electronic visa waiver (EVW)](https://github.com/get-electronic-visa-waiver). You can apply for an EVW from 1 June 2022.
^

to:

^You will not need a visa if you hold a valid [electronic visa waiver (EVW)](https://github.com/get-electronic-visa-waiver). You can apply for an EVW from 1 June 2022.^

Reason:

Markdown is incorrect - there should be no line break after the first ^ and before the second ^.

File: app/flows/check_uk_visa_flow/outcomes/outcome_marriage_visa_nat_direct_airside_transit_visa.erb
Comment (mattclear) [inline app/flows/check_uk_visa_flow/outcomes/outcome_marriage_visa_nat_direct_airside_transit_visa.erb:21]: Change lines 19-21 from:

^
If you want to convert a civil partnership into a marriage, you’ll need to apply for an [electronic visa waiver (EVW)](/get-electronic-visa-waiver) or a [Standard Visitor visa](/standard-visitor).
^

to:

^If you want to convert a civil partnership into a marriage, you’ll need to apply for an [electronic visa waiver (EVW)](/get-electronic-visa-waiver) or a [Standard Visitor visa](/standard-visitor).^

Reason:

Markdown is incorrect - there should be no line break after the first ^ and before the second ^.

File: app/flows/check_uk_visa_flow/outcomes/outcome_transit_leaving_airport.erb
Comment (mattclear) [inline app/flows/check_uk_visa_flow/outcomes/outcome_transit_leaving_airport.erb:14]: Change:

a [Visitor in Transit visa](/transit-visa) if you arrive on a flight and will pass through immigration control before you leave the UK

to:

+ a [Visitor in Transit visa](/transit-visa) 

Reason:

Wording is incorrect. Also bullet markdown is missing.

Comment (mattclear) [inline app/flows/check_uk_visa_flow/outcomes/outcome_transit_leaving_airport.erb:14]: Change:

a [Visitor in Transit visa](/transit-visa) if you arrive on a flight and will pass through immigration control before you leave the UK

to:

+ a [Visitor in Transit visa](/transit-visa) 

Reason:

Wording is incorrect. Also bullet markdown is missing.

**Review by mattclear — CHANGES_REQUESTED** [review]
Hi, just a few minor comments on wording/markdown.

**Review by jon-kirwan — APPROVED** [review]
I've been through various scenarios to get to each of the updated outcome pages and all changes looks good to me 🚀

The only thing I did notice was related to the outcome page for `/flows/check_uk_visa_flow/outcomes/outcome_visit_waiver.erb/` - shown when tested with _Kuwait_. This doesn't seem to be included in the list of pages to be updated but the content is:

> You must either apply for:

  - an [EVW](/get-electronic-visa-waiver)
  - a [Standard Visitor visa](/standard-visitor)

and to be consistent with the other pages maybe it should be updated to:

> You must either apply for:

  - an [electronic visa waiver](/get-electronic-visa-waiver)
  - [a Standard Visitor visa](/standard-visitor) for courses up to 6 months

**Comment by mattclear** [comment]
@jon-kirwan Thanks for flagging that. We should definitely change the link text in the first bullet from 'EVW' to 'electronic visa waiver'.

'for courses up to 6 months' being included on that outcome in the second bullet was something we noticed earlier and queried with the department. Apparently it is correct, so should be left as it is.

**Comment by edwardkerry** [comment]
@mattclear I have made that last change as well to `/flows/check_uk_visa_flow/outcomes/outcome_visit_waiver.erb/`. Are you able to mark your requested changes as complete and we will then merge this in? (the merge button is unavailable until the changes are approved!)

**Review by mattclear — APPROVED** [review]

**Pull request merged**
